src/helpers/OTSubscriberHelper.js

```diff
@@ -37,20 +45,55 @@ const sanitizeSubscriberEvents = (events) => {
   return reassignEvents('subscriber', customEvents, events);
 };
 
+const sanitizeResolution = (resolution) => {
+  switch (resolution) {
+    case null:
+      return { width: MAX_SAFE_INTEGER, height: MAX_SAFE_INTEGER }
+    case '352x288':
```

Comment:
I know this is probably making things a bit more complicated, but do you think that instead of the developer providing a string for the preferredResolution, they specify an object?

Something like:
```
preferredResolution = {
  width: 640,
  height: 480,
};
```

I know this is not consistent with the resolution property when creating a publisher, but we can always try improving :)

Reply:
We can, and it was actually my first approach, but I thought about being consistent with the resolution.
What advantages do you see on using an object? :)

Reply:
I think the object is cleaner and more explicit than using a string.

Reply:
I'll update 👍